### PR TITLE
bite-sized-bears

### PR DESCRIPTION
**Pull request by ryuga-hideki**

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
No merge conflicts.

No flake8 linting errors.

**Comment by SebastiaanZ** [comment]
No flake8 linting errors

**Comment by SebastiaanZ** [comment]
Commits added after the deadline

File: bite-sized-bears/Website/syndication_app/models.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+class Community(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=25)
+    description = models.TextField(blank=True, null=True)
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, related_name='community_owner')
+    admin = models.ManyToManyField(User, related_name='community_admin')
+    subscribers = models.ManyToManyField(User, related_name='community_subscribers')
+    location = models.CharField(max_length=20)
+
+    def get_absolute_url(self):
```
Comment (kosayoda) [inline bite-sized-bears/Website/syndication_app/models.py:30]: Using properties in Python with the `@property` decorator is more idiomatic than having getters.

File: bite-sized-bears/Website/syndication_app/views.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        view.view_initkwargs = initkwargs
+
+        update_wrapper(view, cls, updated=())
+
+        update_wrapper(view, cls.dispatch, assigned=())
+        return view
+
+
+class IndexListView(ListView, View):
+    paginate_by = 25
```
Comment (kosayoda) [inline bite-sized-bears/Website/syndication_app/views.py:66]: `IndexListView` has 25 items per page, while `CommunityListView` has 20 items, and `MostViewedPost` and `MyCommunityListView` have 15 items. A comment with the rationale for having varied pagination sizes would be good.
Alternatively, a constant can be used to unify pagination sizes, eg. `PAGINATION_SIZE = 20`.

**Pull request merged**